apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common/ErrorReporting/ErrorReportingOptions.cs

```diff
@@ -62,15 +48,15 @@ private ErrorReportingOptions(string logName, MonitoredResource resource, Buffer
             RetryOptions = GaxPreconditions.CheckNotNull(retryOptions, nameof(retryOptions));
         }
 
+        // TODO: Consider making this obsolete and creating a new Create now that we don't have the overload ambiguity.
```

Comment:
@jskeet Flagging this TODO.
I say we go for it, even if we start up the new version with some obsolete code.

Reply:
Yes - or just rename it to Create entirely? I don't mind.

Reply:
Renamed entirely on the last commit. I'll merge on green.